### ISSUE TITLE
Deprecation warning python 3.5

### ISSUE THREAD

**Issue by lovetox** [Legrandin/pycryptodome#58]
hi,

i get this on python3.5

```
C:\msys64\mingw32\lib\python3.5\site-packages\Crypto\Util\_raw_api.py:31: PendingDeprecationWarning: the imp module is deprecated in favour of importlib;
see the module's documentation for alternative uses
```

**Comment by Legrandin** [comment]
Thanks. Fixed in c5f64b64b5325032d98e27ed144adc9b59cf5318.

**Comment by drewbanin** [comment]
Hello! I'm also seeing this issue but with pycryptodome v3.4.7 on Python 3.6.4

Just looking at the code, I think there's a bug in the imp vs. importlib logic. On lines 38/39 [here](https://github.com/Legrandin/pycryptodome/blob/master/lib/Crypto/Util/_raw_api.py#L38-L39), it looks like the if-statement will evaluate to `True` for _any_ minor version of Python 3 because the check uses `<= 3` instead of `< 3`. I think it should be something like this, right?

```
if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] <= 3):
```
